src/main/java/com/williamfeliciano/sba_movies/repositories/MovieRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.williamfeliciano.sba_movies.repositories;
+
+import com.williamfeliciano.sba_movies.entities.Movie;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface MovieRepository extends JpaRepository<Movie, Long> {
```

Comment:
Now, you must use this repository in the MoviesService.

Reply:
Can you also write a single unit test for the repository? I don't want any logic to be tested, just put in place an H2 database, mock data from a SQL file and test a single request